lib/metrics.py

```diff
@@ -57,6 +59,23 @@ def emit(self, stats):
             self.fallback_emitter.emit(stats)
 
 
+class LoggingHeartbeatEmitterThread(threading.Thread):
+    def __init__(self, interval):
+        super(LoggingHeartbeatEmitterThread, self).__init__()
```

Comment:
We use python3 now, so specifying the class and self in super is no longer needed, you can simply do:
`super().__init__()`

Reply:
Done